src/main/java/com/denizenscript/depenizen/bukkit/bridges/ViaVersionBridge.java

```diff
@@ -1,12 +1,65 @@
 package com.denizenscript.depenizen.bukkit.bridges;
 
+import com.denizenscript.denizencore.objects.core.ListTag;
+import com.denizenscript.denizencore.tags.PseudoObjectTagBase;
+import com.denizenscript.denizencore.tags.TagManager;
 import com.denizenscript.depenizen.bukkit.Bridge;
 import com.denizenscript.depenizen.bukkit.properties.viaversion.ViaVersionPlayerExtensions;
+import com.viaversion.viaversion.api.Via;
+import com.viaversion.viaversion.api.protocol.version.ProtocolVersion;
 
 public class ViaVersionBridge extends Bridge {
 
+    static class ViaVersionTagBase extends PseudoObjectTagBase<ViaVersionTagBase> {
+
+        public static ViaVersionTagBase instance;
+
+        public ViaVersionTagBase() {
+            instance = this;
+            TagManager.registerStaticTagBaseHandler(ViaVersionTagBase.class, "viaversion", (t) -> instance);
+        }
+
+        @Override
+        public void register() {
+
+            // <--[tag]
+            // @attribute <viaversion.supported_protocol_versions>
+            // @returns ListTag
+            // @plugin Depenizen, ViaVersion
+            // @description
+            // Returns a list of all supported protocol versions on the server.
+            // Versions blocked in the ViaVersion config.yml file will not be shown in this list.
+            // -->
+            tagProcessor.registerTag(ListTag.class, "supported_protocol_versions", (attribute, object) -> {
+                ListTag list = new ListTag();
+                for (Object protocol : Via.getAPI().getSupportedProtocolVersions()) {
+                    int version = ((ProtocolVersion) protocol).getVersion();
+                    list.add(String.valueOf(version));
+                }
+                return list;
+            });
+
+            // <--[tag]
+            // @attribute <viaversion.full_supported_versions>
```

Comment:
I'd probably name this just `supported_versions` (or maybe `supprted_version_names`? if you want to be specific) - `full` sounds like there's a tag that only returns some supported versions/partial version names/versions that are only partially supported or something like that imo.